HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -58,6 +58,13 @@ def customiseFor14833(process):
                 producer.includeME0 = cms.bool(False)
     return process
 
+def customiseFor16670(process):
```

Comment:
Hi @calabria @slava77 -does not a fillDescriptions implementation avoid this function completely? In any case, when will this get propagated to the menu?